source/help/getting-started/organizing-your-sidebar.rst

```diff
@@ -0,0 +1,173 @@
+- All Direct Messages display in a **Direct Messages** category below channels.
+- All Direct Messages are sorted by recent activity.
+
+What can you customize?
+-----------------------
+
+You can customize your sidebar in the following ways:
+
+- Group and order channels into categories customized just for you.
+- Mute and unmute entire categories.
+- Sort category channels manually, alphabetically, or by recent activity.
+- Group unread messages into an **Unreads** category.
+- Sort Direct Messages alphabetically or by recent activity.
+- Control how many Direct Message conversations to display.
+
+Creating custom categories
+--------------------------
+
+Creating your own custom categories enables you to group channels together for easier navigation (e.g. “Design” or “Marketing”).
+
+Categories can be created in two ways:
+
+- Select the **+** symbol to the right of the **History** arrows at the top of the sidebar.
+- Select the **Channel options** icon in the sidebar, then select **Create New Category**.
+
+Next, type a category name, select **Create**, then drag any channels or Direct Messages into this new category. You can also multi-select channels and Direct Messages to drag them together as a group.
+
+Making categories work for you
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Categories are collapsible.**
+
+- When collapsed, only unread channels display to reduce unnecessary scrolling.
+- When expanded, all channels in the category display, including channels with unread messages.
+
+**You can reorder categories.**
+
+- Drag to reorder entire categories to prioritize important conversations. 
+
+**Categories can contain Direct Message conversations.**
+
+- Click and drag Direct Messages into any category. You can also multi-select Direct Messages to drag them together as a group.
+
+    .. image:: ../../images/sidebar_ga.gif
+
+Renaming categories
+^^^^^^^^^^^^^^^^^^^
+
+1. Select the **Category options** icon in the sidebar, then select **Rename Category**.
+2. Type a new category name, then select **Rename**.
+
+Deleting categories
+^^^^^^^^^^^^^^^^^^^
+
+1. Select the **Category options** icon in the sidebar, then select **Delete Category**.
+2. Select **Delete** to confirm or select **X** to cancel.
+
+All channels and Direct Message conversations in the deleted category move back to their default **Channels** and **Direct Messages** categories. Deleting a category never removes you from channels you have joined. 
+
+Organizing channels in categories
+---------------------------------
+
+Once you have categories, you can move channels into your categories to organize your sidebar.
+
+Multi-selecting channels and Direct Messages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Using the Mattermost Web or Desktop App:
+
+- Select sequential channels and/or Direct Messages by pressing and holding SHIFT. 
+- Select non-sequential channels and/or Direct Messages by pressing and holding CMD (for Mac) or CTRL (for Windows/Linux). 
```

Comment:
```suggestion
- Select non-sequential channels and/or Direct Messages then press and hold CMD (for Mac) or CTRL (for Windows/Linux) on your keyboard. 
```

Reply:
In this case, the user needs to press and hold keyboard keys in order to select multiple items at a time.